Automattic/wp-calypso
# Nav Unification: Remove "show wp-admin pages" toggle

Fixes https://github.com/Automattic/wp-calypso/issues/54164
Fixes https://github.com/Automattic/wp-calypso/issues/54405

#### Changes proposed in this Pull Request

Removes the "show wp-admin pages" toggle from the Account Settings since it's no longer needed now that customers have the ability to switch between Calypso and WP Admin using the new "Screen Options" tab.

Before | After
--- | ---
<img width="742" alt="Screen Shot 2021-07-08 at 11 11 48" src="https://user-images.githubusercontent.com/1233880/124941413-0ea6b300-e00b-11eb-8637-06456c85d1eb.png"> | <img width="603" alt="Screen Shot 2021-07-08 at 11 13 23" src="https://user-images.githubusercontent.com/1233880/124941428-11a1a380-e00b-11eb-9faf-9a5fb16fe0ce.png">

#### Testing instructions

- Use the Calypso live link below and go to `/me/account`.
- Make sure the "show wp-admin pages" toggle does not show up.
- Go to `/posts` and select your test site.
- Use the "Screen Options" tab and select "Classic view".
- You'll be in WP Admin, so use again the Calypso live link below to test this PR.
- Switch to your test site.
- Make sure the "Write" button in the top bar redirects to the WP Admin editor.
- You'll be in WP Admin, so use again the Calypso live link below to test this PR.
- Go to `/post` and select your test site.
- Make sure you're redirected to the WP Admin editor.
- Go to `/wp-admin/edit.php`.
- Use the "Screen Options" tab and select "Use WordPress.com view".
- You'll be in Calypso production, so use again the Calypso live link below to test this PR.
- Make sure the "Write" button in the top bar redirects to the Calypso editor.
- Go to `/post` and select your test site.
- Make sure you're redirected to the Calypso editor.
- Go to `/settings/writing` and select your site.
- Make sure the taxonomies management is not visible here (they are accessed from Posts > Categories and Posts > Tags).
